packages/jira-adapter/src/filters/forms/forms_types.ts

```diff
@@ -0,0 +1,227 @@
+/*
+*                      Copyright 2023 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import { ObjectType, ElemID, BuiltinTypes, CORE_ANNOTATIONS, ListType } from '@salto-io/adapter-api'
+import Joi from 'joi'
+import { elements as adapterElements } from '@salto-io/adapter-components'
+import { createSchemeGuard } from '@salto-io/adapter-utils'
+import { JIRA, FORM_TYPE } from '../../constants'
+
+export type DetailedFormDataResponse = {
+    id: number
+    updated: string
+    uuid: string
+    publish: {}
+    design: {
+      settings: {
+        templateId: string
+        name: string
+        submit: {
+          lock: boolean
+          pdf: boolean
+        }
+        templateFormUuid: string
+      }
+      questions: {}
+      sections: {}
+      conditions: {}
+    }
+}
+
+type FormResponse = {
+  id: number
+  name: string
+}
+
+export type FormsResponse = {
+  data: FormResponse[]
+}
+
+export const FORMS_RESPONSE_SCHEME = Joi.object({
+  data: Joi.array().items(Joi.object({
+    id: Joi.number().required(),
+    name: Joi.string().required(),
+  }).unknown(true).required()),
+}).unknown(true).required()
+
+export const DETAILED_FORM_RESPONSE_SCHEME = Joi.object({
+  uuid: Joi.string().required(),
+  design: Joi.object({
+    settings: Joi.object({
+      templateId: Joi.number().required(),
+      name: Joi.string().required(),
+      submit: Joi.object({
+        lock: Joi.boolean().required(),
+        pdf: Joi.boolean().required(),
+      }).required(),
+      templateFormUuid: Joi.string().required(),
+    }).unknown(true).required(),
+    questions: Joi.object().unknown(true).required(),
+    sections: Joi.object().unknown(true).required(),
+    conditions: Joi.object().unknown(true).required(),
+  }).unknown(true).required(),
+}).unknown(true).required()
+
+
+export const createFormType = (): {
+    formType: ObjectType
+    subTypes: ObjectType[]
+  } => {
+  const FormSubmitType = new ObjectType({
```

Comment:
All ObjectType variables here should start with lowercase as this is a variable and not a typescript type 
(also relevant for all other Object types in this function) 
```suggestion
  const formSubmitType = new ObjectType({
```

Reply:
oh, I miss understand this. Thanks